### ISSUE TITLE
CUDA out of memory run time error after playing a while with GPT-neo model

### ISSUE THREAD

**Issue by Lazorr** [cloveranon/Clover-Edition#181]
Game starts fine but after my story gets longer I get this error and I'm unable to progress further. If I load the save it appears immediately again. This happens on RTX 2080.

RuntimeError: CUDA out of memory. Tried to allocate 308.00 MiB (GPU 0; 8.00 GiB total capacity; 5.79 GiB already allocated; 51.50 MiB free; 6.28 GiB reserved in total by PyTorch)

A fatal error has occurred.

**Comment by MikkoMMM** [comment]
A duplicate of #167. The solution, for now, is to change the line
`history-gpt-neo = 2048`
in config.ini to having something a bit lower as the value.
`history-gpt-neo = 2000`
might already be enough.

**Comment by Thebombuknow** [comment]
Yeah, this happens when you run out of VRAM. Something else you can do is use CUDA unified memory, which allows TF to use both VRAM and system ram. It’ll definitely slow down the processing, but it’ll at least run.
